### ISSUE TITLE
The parser is failing to parse codechef problems on MS Edge Browser. 

### ISSUE THREAD

**Issue by pawan-nirpal-031** [jmerle/competitive-companion#266]
The problem parser is unable to parse codechef's new problem format, here's what it says. 
```
Something went wrong while running Competitive Companion's CodeChefNewProblemParser. Open the browser console to see the error. Please open an issue at https://github.com/jmerle/competitive-companion/issues if you think this is a bug.
```

The browser log says : 
```
content.js:1 
        
       TypeError: Cannot read properties of null (reading 'textContent')
    at CodeChefNewProblemParser.parse (common.js:1:297950)
    at l (content.js:1:320)
    at content.js:1:1220
l @ content.js:1
content.js:1 
        
       TypeError: Cannot read properties of null (reading 'textContent')
    at CodeChefNewProblemParser.parse (common.js:1:297950)
    at l (content.js:1:320)
    at e (content.js:1:1567)
    at common.js:1:248433
l @ content.js:1
content.js:1 
        
       TypeError: Cannot read properties of undefined (reading 'textContent')
    at CodeChefOldProblemParser.parse (common.js:1:299178)
    at l (content.js:1:320)
    at e (content.js:1:1567)
    at common.js:1:248433
l @ content.js:1
content.js:1 
        
       TypeError: Cannot read properties of null (reading 'textContent')
    at CodeChefNewProblemParser.parse (common.js:1:297950)
    at l (content.js:1:320)
    at content.js:1:1220
```

**Comment by devhiteshk** [comment]
** CLICK old problems page ** then parse it will work

**Comment by jmerle** [comment]
This is a duplicate of https://github.com/jmerle/competitive-companion/issues/250#issuecomment-1186421216, which was fixed in 3829147a6c33ff2b1f8a874bce111598172dcc4b, which was released in 2.35.0 half an hour ago. The update has already been published on GitHub Releases and Mozilla's AMO, and is pending review on the Chrome Web Store.
